src/store/models/taro.ts

```diff
@@ -66,7 +139,9 @@ const taroModel: TaroModel = {
           .filter(n => n.status === Status.Started)
           .map(async n => {
             try {
-              await actions.getAssets(n);
+              await (async () => {
+                actions.getAllInfo(n);
+              })();
```

Comment:
You shouldn't need this `async` IIFE if you make the change I mentioned above and the mining is already occuring in the background. This listener will also run in the background.

Reply:
Done